### PR TITLE
Remove generic text from How To Test page

### PR DESCRIPTION
**Pull request by joe-watkins**
Removed generic text from the How To Test page to simplify.

### CODE REVIEW

**Comment by joe-watkins** [comment]
@tritland @silverli  This PR just basically removes the big chunk of text below the cards. It felt a little obligatory and felt strange just dangling below the new cards Charlie added.

File: how-to-test.md [hunk 1]
```diff
@@ -4,9 +4,3 @@ title: Websites
 seoTitle: How to test accessibility
 permalink: /how-to-test/
 ---
-
-Accessibility testing is important because it helps to ensure that everyone can use a website or application, regardless of their disability. This includes people who are blind, deaf, have low vision, or have mobility impairments.
```
Comment (silverli) [inline how-to-test.md:8]: do you think this text would fit on the "Not sure it's an issue page?"

**Review by silverli — CHANGES_REQUESTED** [review]
One question!

**Review by tritland — DISMISSED** [review]

File: how-to-test.md (hunk 1 above)
Comment (joe-watkins) [reply how-to-test.md:8]: @silverli Good idea! I've added it to the Not Sure page ;) https://github.com/tmobile/magentaA11y/pull/64/commits/f5d30ca9555d2b6e585d52de383058bdca0b4bfb

**Review by tritland — APPROVED** [review]

File: how-to/not-sure-if-it-is-an-issue.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ permalink: /how-to-test/not-sure-if-it-is-an-issue/
 
 When conducting accessibility testing, testers may come across barriers that could be accessibility-related issues, or they could be unrelated. If you're unsure whether the problem you've identified is a genuine accessibility issue, the following steps can help you verify it.
 
+Accessibility testing is important because it helps to ensure that everyone can use a website or application, regardless of their disability. This includes people who are blind, deaf, have low vision, or have mobility impairments.
```
Comment (silverli) [inline how-to/not-sure-if-it-is-an-issue.md:9]: great!

**Review by silverli — APPROVED** [review]
approved!

**Pull request merged**